pkg/util/hostname.go

```diff
@@ -125,6 +125,21 @@ func GetHostname() (string, error) {
 		log.Debug("Unable to get hostname from GCE: ", err)
 	}
 
+	// FQDN
+	log.Debug("GetHostname trying FQDN/`hostname -f`...")
+	name, err = getSystemFQDN()
+	if err == nil {
+		if config.Datadog.GetBool("hostname_fqdn") {
+			return name, nil
```

Comment:
I think we actually want to be careful about returning the value here already, see `isContainerized` logic below that we probably want to keep before we check for `hostname -f`